site/public/css/gradeable-notebook.css

```diff
@@ -110,7 +110,9 @@ img {
     border: 1px solid var(--standard-light-gray) !important;
     border-radius: 4px !important;
     vertical-align: middle !important;
-    resize: both;
+    width: 100% !important;
+    resize: vertical;
```

Comment:
For a short answer, it may be useful to be able to extend horizontally, and it is definitely not useful to be able to extend vertically.